src/resources/views/corporation/ledger/reprocessingfeebymonth.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('web::corporation.ledger.layouts.view', ['sub_viewname' => 'bountyprizesbymonth'])
+
+@section('title', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
```

Comment:
Is it really bounty ? You should update the call from `seat.bountyprizesbymonth` to something related with your view. This is the caption appearing into your browser tab. You will also have to set it into this file https://github.com/eveseat/web/blob/master/src/lang/en/seat.php#L363